app/views/components/_notice.html.erb

```diff
@@ -3,8 +3,15 @@
     description_text ||= ""
     description_govspeak ||= ""
   %>
+
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
+
+    <% if description_text.present? || description_govspeak.present? %>
```

Comment:
We've definitely done this for other components, so I see no reason why it wouldn't work here. Was there any particular reason why you chose to use `.present?` when you implemented the component?